### ISSUE TITLE
Can't get it working on Laravel 4.2

### ISSUE THREAD

**Issue by bcalik** [backup-manager/laravel-4#6]
Hello,

I've been using the heybigname/backup-manager package in a project with Laravel 4.2 for a long time. Which has been working great. I noticed that the repository has been moved. So I'm trying to update it. But I can't get it work.

I get this error whatever I try to backup.
`Could not find configuration for connection local`

Here is the `app/config/backup-manager.php` file:

``` php
return [
    'local' => [
        'type' => 'Local',
        'root' => '/',
    ],
    ...
]
```

I also tried to put this file into `app/config/packages/backup-manager/laravel` folder.

What I'm doing wrong? Previous repo was working fine.

**Comment by ShawnMcCool** [comment]
The old repository is still there at heybigname/backup-manager you might be able to use that until we get to the bottom of this. I have been traveling today and haven't been able to dig into this.

**Comment by ShawnMcCool** [comment]
Hi @bcalik, force an upgrade to ^0.2.9 and place your config in app/config/backup-manager.php

Should work.

**Comment by ShawnMcCool** [comment]
Ok, just released a new version 1.0 of this repo that has been tested in my laravel 4 and 5 test setups.
